test/IntegrationTests/Helpers/EnvironmentHelper.cs

```diff
@@ -207,9 +207,9 @@ public static bool IsRunningOnCI()
             environmentVariables["OTEL_DOTNET_AUTO_INCLUDE_PROCESSES"] = Path.GetFileName(processToProfile);
         }
 
-        string integrations = GetIntegrationsPath();
         environmentVariables["OTEL_DOTNET_AUTO_HOME"] = GetNukeBuildOutput();
-        environmentVariables["OTEL_DOTNET_AUTO_INTEGRATIONS_FILE"] = integrations;
+
+        environmentVariables["OTEL_DOTNET_AUTO_INTEGRATIONS_FILE"] = Environment.GetEnvironmentVariable("OTEL_DOTNET_AUTO_INTEGRATIONS_FILE") ?? GetIntegrationsPath();
```

Comment:
I think that many code paths will not be able to retrieve the value with the `Environment.GetEnvironmentVariable` call because this method is called after the method that clears that environment variable. `InstrumentedProcessHelper.StartInstrumentedProcess` is one example.

Reply:
The `environmentHelper` is built before that pointing in the code, that's why it works as intended. That said the code is in need of a refactor to simplify and make things easier to maintain.

Reply:
#1155